src/Pomodoro.Web/Components/Tasks/TaskListTabs.razor.css

```
@@ -1,25 +1,28 @@
 }
 
 .lt {
     display: flex;
     align-items: center;
     gap: 6px;
-    padding: 4px 10px;
+    padding: 4px 10px 5px;
     font-size: 12px;
     border: none;
     background: transparent;
     color: var(--color-text-secondary);
```

Stylelint (17.14.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)